### ISSUE TITLE
♿ Accessibility

### ISSUE THREAD

**Issue by thiemeljiri** [silencesys/silentbox#52]
Hi,
the gallery generated via this plugin is completely inaccessible. You cannot use keyboard at all. The items should be standard links with images, so that they are included in tab order of the page.

BR,
Jiri

**Comment by silencesys** [comment]
Hey,

thanks for letting me know. I don't remember why I used `divs` instead of anchors, as it seems nothing is broken when I change them. Thus it should be fixed in 2.4.4.
